draganddrop/draganddrop/MainPage.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
This sample does not follow the standard sample template. It will be destroyed the next time there is a template update.